packages/plugin-commands-env/test/env.test.ts

```diff
@@ -25,3 +26,37 @@ test('install node', async () => {
   const dirs = fs.readdirSync(path.resolve('nodejs'))
   expect(dirs).toEqual(['16.4.0'])
 })
+
+test('install node by version range', async () => {
+  tempDir()
+
+  await env.handler({
+    bin: process.cwd(),
+    global: true,
+    pnpmHomeDir: process.cwd(),
+    rawConfig: {},
+  }, ['use', '6'])
+
+  const { stdout } = execa.sync('node', ['-v'], {
+    env: {
+      [PATH]: `${process.cwd()}${path.delimiter}${process.env[PATH] as string}`,
+    },
+  })
+  expect(stdout.toString()).toBe('v6.17.1')
+
+  const dirs = fs.readdirSync(path.resolve('nodejs'))
+  expect(dirs).toEqual(['6.17.1'])
```

Comment:
could users download any Node.js version?

Reply:
Any of those that are in this index: https://nodejs.org/download/release/index.json

Reply:
I see, the point is being able to download any Node.js version https://github.com/pnpm/pnpm/discussions/3434 but the bundle one is something already predefined in build time.

Reply:
Yes, the bundled one is static. The bundled one is only used to run the pnpm CLI